package.json

```
@@ -27,11 +27,14 @@
   ],
   "main": "dist/index.js",
   "module": "src/index.js",
+  "types": "types/index.d.ts",
   "files": [
     "dist",
+    "types/index.d.ts",
     "src/index.js"
   ],
   "dependencies": {
+    "@types/hast": "^2.3.2",
```

where possible widen the version range to allow npm to deduplicate minor/patch releases

```suggestion
    "@types/hast": "^2.0.0",
```